src/components/views/sul-study-place/study-place-today-hours-table.tsx

```diff
@@ -0,0 +1,35 @@
+"use client";
+
+import {ClockIcon} from "@heroicons/react/24/outline";
+import useTodayLibraryHours from "@/lib/hooks/useTodayLibraryHours";
+import CachedClientFetch from "@/components/utils/cached-client-fetch";
+
+const StudyPlaceHours = ({hoursId}: { hoursId: string }) => {
+  return (
+    <CachedClientFetch>
+      <StudyPlaceHoursComponent hoursId={hoursId}/>
+    </CachedClientFetch>
+  )
+}
+
+const StudyPlaceHoursComponent = ({hoursId}: { hoursId: string }) => {
+  const hours = useTodayLibraryHours(hoursId);
+  if (!hours) {
+    return null;
+  }
+
+  const {closedAllDay, isOpen, openingTime, closingTime, afterClose} = hours;
+  const openUntil = "Open until ";
```

Comment:
@pookmish I'd like to put a class on this Open but we can't have html in the variable and string them together like the olden days.  What is a better way to think about this challenge?

Reply:
put the variable into the markup:
`<span className="foobar">{openUntil}</span><span> More Stuff</span>`

Reply:
You should always use the tags and markup in the component, not in variables.